.github/workflows/build-website.yml

```diff
@@ -26,10 +27,19 @@ jobs:
         with:
           node-version: "16"
 
-      - name: Check Translations
+      - name: Install dependencies
```

Comment:
You probably wanted to revert this name change too, right?

Reply:
No, this job does everything: run prettier, check translations, build the site and upload the artifacts. That's why we need to install dependencies. I just made it a separate step (it was done in the check translations step previously)

Reply:
Ah, I totally missed what was going on here, sorry! I thought you renamed the entire job to "Install dependencies" :)

In my defence, it was very early in the morning :D